Cargo.toml

```diff
@@ -1089,7 +1089,8 @@ polkadot-subsystem-bench = { path = "polkadot/node/subsystem-bench" }
 polkadot-test-client = { path = "polkadot/node/test/client" }
 polkadot-test-runtime = { path = "polkadot/runtime/test-runtime" }
 polkadot-test-service = { path = "polkadot/node/test/service" }
-polkavm = { version = "0.9.3", default-features = false }
+polkavm = { version = "0.16.0", default-features = false }
+polkavm-common = "0.16.0"
```

Comment:
`polkavm-common` is a *private* PolkaVM crate and should not be used in dependencies

Reply:
I checked where I needed it:

```
$ git -P grep polkavm_common
substrate/client/executor/common/src/error.rs:impl From<polkavm_common::program::ProgramParseError> for WasmError {
substrate/client/executor/common/src/error.rs:	fn from(error: polkavm_common::program::ProgramParseError) -> Self {
```
Related diff:

```
-impl From<polkavm::ProgramParseError> for WasmError {
-       fn from(error: polkavm::ProgramParseError) -> Self {
+impl From<polkavm_common::program::ProgramParseError> for WasmError {
+       fn from(error: polkavm_common::program::ProgramParseError) -> Self {
                WasmError::Other(error.to_string())
        }
 }
```
I did this because I got compilation errors. How can I reach that struct?

Reply:
https://docs.rs/polkavm/latest/polkavm/?search=ProgramParseError